task/src/main/java/no/nav/vedtak/felles/prosesstask/impl/RunTask.java

```diff
@@ -91,10 +93,12 @@ protected void runTaskAndUpdateStatus(Connection conn, ProsessTaskEntitet pte, P
                 return;
             }
 
-            pickAndRun.dispatchWork(pte);
+            OtelUtil.wrapper().span("TASK " + taskType.value(), OtelUtil.taskAttributter(pte.tilProsessTask()), () -> {
```

Comment:
Trenger du både her og i ProsessTaskHandlerRef? 
Dispatchwork kaller ganske direkte ned til doTask ...

Reply:
Skal du beholde denne - så kan du la dispatchWork ta inn ProsessTaskData - så du slipper to kall til pte.tilProsessTask()